zebra-consensus/src/primitives/groth16.rs

```diff
@@ -102,7 +122,10 @@ pub static SPEND_VERIFIER: Lazy<
             (|item: Item| {
                 Verifier::verify_single_spawning(
                     item,
-                    &GROTH16_PARAMETERS.sapling.spend_prepared_verifying_key,
+                    &GROTH16_PARAMETERS
+                        .sapling
+                        .spend_prepared_verifying_key
+                        .inner(),
```

Comment:
This is one of the functions added to `sapling-crypto`: https://github.com/oxarbitrage/sapling-crypto/blob/add-verifying-key-methods/src/circuit.rs#L593

Reply:
Do we want to copy the `parse_parameters` function from `zcash_proofs` and update the `SaplingParameters.spend_prepared_verifying_key` field type to `groth16::PreparedVerifyingKey<Bls12>` instead? Or is there an alternative to Zebra's `verify_proof()` function that accepts the `zcash_proofs` `PreparedSpendVerifyingKey` type?

Reply:
I will definitely try any alternative but I am not understanding this approach. Can you explain more or differently ?

Reply:
I'll open a suggestion PR, note that @upbqdn would prefer we wait for the next version, and if we're okay waiting that could work too, but my concern is that we'll have to wait for the next zcashd release so there aren't dependency minor/patch version conflicts.

Reply:
Opened https://github.com/ZcashFoundation/zebra/pull/8548 with the changes I had in mind.